corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -306,6 +306,34 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         },
     });
 
+    const CaseTileGroupedView = CaseTileView.extend({
+        tagName: "div",
+        className: "formplayer-request list-cell-wrapper-style case-tile-group",
+        template: _.template($("#case-tile-grouped-view-item-template").html() || ""),
+        templateContext: function () {
+            const dict = CaseTileGroupedView.__super__.templateContext.apply(this, arguments);
+            dict['groupHeaderRows'] = this.options.groupHeaderRows;
+            dict['headerData'] = this.options.model.get('data').slice(0, this.options.groupHeaderRows);
```

Comment:
This isn't quite right, this takes the first N `fields` however we want the first N `rows` (each row may have more than 1 field). For example, using the "Title row, second row with two cells, third row, and map" template with 2 header rows would require selecting the first 3 fields.

I'm not sure that we can guarantee that the fields will appear in their 'row' order. 

What I think we need to do is to use the grid configuration for each tile field to determine if it is part of the 'header' or not.

Reply:
In the interest of time I took a stab at this: 83d92849fac

This takes all fields where `gridY < headerRows` i.e. all fields that are (at least partially) in the header rows.

I'm ignoring `gridHeight` (which could make a field span beyond the header) for now under the assumption that we won't configure things that way.

Reply:
Ah dang, thank you for the help implementing that as well. The first "N" row behavior is what i understood coming out our discussion previously but it somehow shifted while I was implementing.